ksql-engine/src/main/java/io/confluent/ksql/util/KafkaTopicClient.java

```diff
@@ -74,11 +73,12 @@ void createTopic(
   Map<String, TopicDescription> describeTopics(Collection<String> topicNames);
 
   /**
-   * [warn] synchronous call to get the response
+   * [warn] Asynchronous call to get the response
```

Comment:
This is a synchronous call. It blocks until it gets a successful response or will throw an exception if it didn't get a successful response after 5 retries.